src/fit.py

```diff
@@ -29,3 +54,36 @@
 ##   The relevant command is described here: https://matplotlib.org/stable/api/_as_gen/matplotlib.figure.Figure.savefig.html#matplotlib.figure.Figure.savefig
 ## - Note it is often convenient to not commit generated files to repositories, but their recipe instead!
 ##   (That is why there is a `.gitignore` file in the `plots/` folder.)
+
+def simulate(v: float, alpha: float, t_max: float) -> tp.Tuple[np.ndarray, np.ndarray]:
+    t = np.linspace(0, t_max, 20)
+    vx = v * np.cos(alpha)
+    vy = v * np.sin(alpha)
+    x = vx * t
+    y = vy * t - 0.5 * EARTH_GRAVITY * t**2
+    return x, y
+
+
+def plot(x: np.ndarray, y: np.ndarray, name: str = None) -> plt.Figure:
+    v, alpha = fit(x, y)
+    x_sim, y_sim = simulate(v, alpha, 4)
```

Comment:
the `4` is of course not always true :upside_down_face: again if you use `model` it is easier to generealize - because you can just depend on e.g. `max(x)`